src/express.ts

```diff
@@ -156,8 +154,18 @@ export function createRouter(sofa: Sofa): Middleware {
     const slicedUrl = url.slice(sofa.basePath.length);
     const obj = router.find(req.method as Trouter.HTTPMethod, slicedUrl);
     try {
+      const contextValue = isContextFn(sofa.context)
+        ? await sofa.context({ req, res })
+        : sofa.context;
+      const subscriptionManager = new SubscriptionManager(sofa, contextValue);
```

Comment:
I think this is now created on every request. It was created once on bootstrap before.

Reply:
Hm, alternatively routes may pass context to subscriptionManager methods.

Reply:
@ardatan Is it good to go now?

Reply:
Yes! Thanks!